src/Components/EnvironmentPicker/EnvironmentPicker.tsx

```diff
@@ -1,585 +1,518 @@
 import {
-    ComboBox,
     DefaultButton,
     Dialog,
     DialogFooter,
     DialogType,
     FontIcon,
     FontSizes,
-    IComboBoxOption,
-    IComboBoxStyles,
```

Comment:
It looks really bad when we show entries without a URL property. I think they should be hidden from teh list instead of showing this error message. 
@darsney thoughts?
![image](https://user-images.githubusercontent.com/57726991/184447359-0dbd68b6-0cf0-4c32-8e36-7cefc1cf26d5.png)

Reply:
You can't use it anyways so it just looks bad and doesn't help the user.
![image](https://user-images.githubusercontent.com/57726991/184447795-3076f7aa-a02b-40c0-bcc1-319f1d66b303.png)

Reply:
yeah we should remove these